saleor/graphql/checkout/mutations/order_from_checkout_create.py

```diff
@@ -0,0 +1,95 @@
+import graphene
+from django.core.exceptions import ValidationError
+
+from ....checkout.checkout_cleaner import validate_checkout
+from ....checkout.complete_checkout import create_order_from_checkout
+from ....checkout.error_codes import OrderFromCheckoutCreateErrorCode
+from ....checkout.fetch import fetch_checkout_info, fetch_checkout_lines
+from ....core import analytics
+from ....core.exceptions import GiftCardNotApplicable, InsufficientStock
+from ....core.permissions import CheckoutPermissions
+from ....discount.models import NotApplicable
+from ...core.descriptions import PREVIEW_FEATURE
+from ...core.mutations import BaseMutation
+from ...core.types.common import OrderFromCheckoutCreateError
+from ...order.types import Order
+from ..types import Checkout
+from ..utils import prepare_insufficient_stock_checkout_validation_error
+
+
+class OrderFromCheckoutCreate(BaseMutation):
+    order = graphene.Field(Order, description="Placed order.")
+
+    class Arguments:
+        id = graphene.ID(
+            required=True,
+            description="ID of a checkout that will be converted to an order.",
+        )
+        clear_checkout = graphene.Boolean(
```

Comment:
Maybe `remove_checkout` would be better? To me "clear" sounds like keep the checkout instance but clear it = remove some data from it? But maybe it's fine, I won't argue about it 😄